packages/patternfly-4/content/get-started/gettingstarted-design.md

```diff
@@ -6,7 +6,29 @@ path: "/get-started/designers"
 
 <p>Before you begin, check out <a href="/get-started/about">About PatternFly 4</a> to get acquainted with the basic elements of the design system.</p>
 
-<h1 class="pf-c-title pf-m-3xl pf-u-mt-xl">Designing with PatternFly 4</h1>
+<h1 class="pf-c-title pf-m-3xl pf-u-mt-xl">Get set up</h1>
+
+<h1 class="pf-c-title pf-m-2xl pf-u-mt-xl">Download and install the PatterFly 4 design kit</h1>
+
+<p>The design kit is a collection of assets that make it easy for you to create high-fidelity design mockups using PatternFly 4 components.</p>
+
+<p>The design kit works with Sketch libraries and templates. To use it, you'll need to have <div class="link"><a href="https://www.sketch.com/">Sketch</a> version 47 or above.</p>
```

Comment:
"The design kit includes Sketch libraries and templates" I think makes more sense here